build/build-bundle.js

```diff
@@ -33,6 +33,11 @@ const gatherers = LighthouseRunner.getGathererList()
 const locales = fs.readdirSync(__dirname + '/../lighthouse-core/lib/i18n/locales/')
     .map(f => require.resolve(`../lighthouse-core/lib/i18n/locales/${f}`));
 
+// HACK: manually include the lighthouse-plugin-publisher-ads audits.
+/** @type {Array<string>} */
+// @ts-ignore
+const pubAdsAudits = require('lighthouse-plugin-publisher-ads/plugin.js').audits.map(a => a.path);
```

Comment:
or whatever is a good way to get this list...readdir doesn't work cause there's more than just the audits in that directory